src/plugins/conditionals/conditionals.c

```diff
@@ -0,0 +1,413 @@
+/**
+ * @file
+ *
+ * @brief Source for conditionals plugin
+ *
+ * @copyright BSD License (see doc/COPYING or http://www.libelektra.org)
+ *
+ */
+
+
+#ifndef HAVE_KDBCONFIG
+# include "kdbconfig.h"
+#endif
+
+#include <string.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <kdberrors.h>
+#include <regex.h>
+#include <ctype.h>
+#include <math.h>
+#include "conditionals.h"
+
+typedef enum{EQU, NOT, LT, LE, GT, GE}Comparator;
+
+static int isNumber(const char *s)
+{
+	char *ptr = (char *)s;
+	int ret = 1;
+	while(*ptr != '\0')
+	{
+		if(*ptr == '.' && ret == 1)
```

Comment:
What about multiple `.`? Why not use `strtol` or similar to find out if its a number?

atof is used short afterwards anyway.